test/local_entry_test.rb

```diff
@@ -41,56 +41,71 @@ def test_read_local_entry_from_non_zip_file
   end
 
   def test_read_local_entry_from_truncated_zip_file
-    zipFragment = ''
-    ::File.open(TestZipFile::TEST_ZIP2.zip_name) { |f| zipFragment = f.read(12) } # local header is at least 30 bytes
-    zipFragment.extend(IOizeString).reset
+    fragment = ''
+    # local header is at least 30 bytes
+    ::File.open(TestZipFile::TEST_ZIP2.zip_name) { |f| fragment = f.read(12) }
+
+    fragment.extend(IOizeString).reset
     entry = ::Zip::Entry.new
-    entry.read_local_entry(zipFragment)
+    entry.read_local_entry(fragment)
     raise 'ZipError expected'
   rescue ::Zip::Error
   end
 
   def test_write_entry
-    entry = ::Zip::Entry.new('file.zip', 'entryName', 'my little comment',
+    entry = ::Zip::Entry.new('file.zip', 'entry_name', 'my little comment',
```

Comment:
I think this is fine, but is this change actually required? (There are a few more like it below.)

Reply:
I was just being comprehensive with getting rid of camel case.